alphadia/workflow/peptidecentric.py

```diff
@@ -549,8 +549,8 @@ def filter_dfs(self, precursor_df, fragments_df):
             by="correlation", ascending=False
         )
         # Determine the number of fragments to keep
-        min_fragments, max_fragments = 500, 5000
-        min_correlation = 0.7
+        min_fragments, max_fragments = 500, self.config["calibration"]["max_fragments"]
```

Comment:
mb add a TODO repeating what you mentioned in the description (that `min_fragments` could possibly be removed)